src/components/dfp-ad-wrapper.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { createDfpAdComponent } from "./dfp-ad";
+
+export function DfpAdWrapper(props) {
+  const DfpAd = createDfpAdComponent(props);
```

Comment:
This is bad. It creates a new class on every render. What is it there for?

Reply:
Previously this logic was in app, now want to move it out so did it but yes, let us think of something else